irida_staramr_results/cli.py

```diff
@@ -1,10 +1,14 @@
 import sys
```

Comment:
The make file is currently broken.
yaml needs to be added to the requirements.txt file.

Reply:
Done.